docs/configuration/git-providers/bitbucket-enterprise.md

```diff
@@ -1,6 +1,11 @@
 # Bitbucket Stash
 
 Set your configuration values for your Bitbucket instance on the `values.yaml` file.
+
+NOTE: Since bitbucket uses OAuth1, you'll need to create a key pair to sign and validate the requests between codacy and the bitbucket enterprise instance.
```

Comment:
Maybe take the opportunity here to rename this Bitbucket Server

Reply:
Yes, although there is both Bitbucket Server and Bitbucket Data Center in their self-hosted offering. :thinking: